From ac13a54647f4cf68ec967c918c7ce0b149e0537e Mon Sep 17 00:00:00 2001
From: Paresh Bhagat <p-bhagat@ti.com>
Date: Thu, 27 Apr 2023 11:27:04 +0530
Subject: [PATCH 7/7] arm64: dts: am625-base-board: Reserve memory for
 jailhouse

This has to be in sync with jailhouse cell configurations.

Upstream-Status: Pending

Signed-off-by: Matt Ranostay <mranostay@ti.com>
Signed-off-by: Paresh Bhagat <p-bhagat@ti.com>
---
 .../dts/ti/k3-am625-base-board-jailhouse.dtso | 30 +++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 arch/arm64/boot/dts/ti/k3-am625-base-board-jailhouse.dtso

diff --git a/arch/arm64/boot/dts/ti/k3-am625-base-board-jailhouse.dtso b/arch/arm64/boot/dts/ti/k3-am625-base-board-jailhouse.dtso
new file mode 100644
index 000000000000..fd59f525bacf
--- /dev/null
+++ b/arch/arm64/boot/dts/ti/k3-am625-base-board-jailhouse.dtso
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (C) 2022 Texas Instruments Incorporated - http://www.ti.com/
+ */
+
+/dts-v1/;
+/plugin/;
+
+&reserved_memory {
+	#address-cells = <2>;
+	#size-cells = <2>;
+
+	/*
+	 * Reserve for Jailhouse hypervisor firmware and inmates.
+	 * Must be in sync with jailhouse cell configurations.
+	 */
+	hyp_mem: jailhouse@dfa00000 {
+		reg = <0x0 0xdfa00000 0x0 0x20600000>;
+		alignment = <0x1000>;
+		no-map;
+	};
+};
+
+//&main_uart1 {
+//	status = "disabled";
+//};
+
+//&sdhci0 {
+//	status = "disabled";
+//};
-- 
2.17.1

